hazelcast/src/main/java/com/hazelcast/nio/serialization/DefaultData.java

```diff
@@ -24,11 +24,12 @@
 @edu.umd.cs.findbugs.annotations.SuppressWarnings("EI_EXPOSE_REP")
 public final class DefaultData implements Data {
 
+    public static final int DATA_OFFSET = 5;
```

Comment:
Can you also make `TYPE_OFFSET` and `PARTITION_HASH_BIT_OFFSET` public and keep them in order again? To me, seeing them in order makes easier to read/understand. 

```java
public static final int TYPE_OFFSET = 0;
public static final int PARTITION_HASH_BIT_OFFSET = 4;
public static final int DATA_OFFSET = 5;
```